diff --git a/node_modules/@remix-pwa/worker-runtime/dist/src/utils/response.js b/node_modules/@remix-pwa/worker-runtime/dist/src/utils/response.js
index 0ad7268..0cf7c52 100644
--- a/node_modules/@remix-pwa/worker-runtime/dist/src/utils/response.js
+++ b/node_modules/@remix-pwa/worker-runtime/dist/src/utils/response.js
@@ -1,10 +1,9 @@
-import { json } from '@remix-run/server-runtime/dist/responses.js';
 /**
  * Converts an error response to a JSON response.
  */
 export function errorResponseToJson(errorResponse) {
     // @ts-expect-error
-    return json(errorResponse.error || { message: 'Unexpected Server Error' }, {
+    return Response.json(errorResponse.error || { message: 'Unexpected Server Error' }, {
         status: errorResponse.status,
         statusText: errorResponse.statusText,
         headers: {
